src/wizard/dart/index.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Dart
+doc_link: https://docs.sentry.io/platforms/dart/
+support_level: production
+type: framework
+---
+
+Get the SDK from [pub.dev](https://pub.dev/packages/sentry) by adding the following to your `pubspec.yaml`:
+
+<PlatformContent includePath="getting-started-install" />
+
+Import `Sentry` and initialize it:
+
+<PlatformContent includePath="getting-started-config" />
+
+Capture a test exception:
+
+<PlatformContent includePath="getting-started-verify" />
+
+### Next steps
+
+Check out the [Dart documentation](https://sentry-docs-git-ref-flutter-v4.sentry.dev/platforms/dart/) to learn about configuration options available.
```

Comment:
```suggestion
Check out the [Dart documentation](http://docs.sentry.io/platforms/dart/) to learn about configuration options available.
```

Reply:
CI is not happy because `http://docs.sentry.io/platforms/dart/` does not exist yet

Reply:
I removed it, the wizard adds automatically a Note + button for docs